packages/flutter/test/material/app_bar_test.dart

```diff
@@ -217,7 +217,7 @@ void main() {
     final Finder title = find.byKey(titleKey);
     expect(tester.getTopLeft(title).dx, 72.0);
     // The toolbar's contents are padded on the right by 4.0
-    expect(tester.getSize(title).width, equals(800.0 - 72.0 - 4.0));
+    expect(tester.getSize(title).width, equals(800.0 - 4.0 - 16.0 - 56.0 - 16.0));
```

Comment:
You could improve this further by explaining what the 16s and the 56 mean (you could even use kToolbarHeight where that's appropriate).

800.0 - 4.0 - 16.0 - 56.0 - 16.0 = 708
800.0 - 72.0 - 4.0 = 724
What changed?